packages/sdk/src/base/transport/axios-transport.ts

```diff
@@ -143,7 +143,9 @@ export class AxiosTransport implements ITransport {
 
 	private createRequestConfig(config: AxiosRequestConfig): AxiosRequestConfig {
 		let token = this._storage.auth_token;
-		if (!token) {
+		let validUntil = this._storage.auth_valid_until;
```

Comment:
This will break, did you miss the changes in the storage interface?

Reply:
Yup :) Will fix.